var/spack/repos/builtin/packages/rust-indexmap/package.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack.package import *
+
+
+class RustIndexmap(CargoPackage):
+    """A hash table with consistent order and fast iteration; access items by key or sequence index"""
+
+    # FIXME: Add a proper url for your package's homepage here.
```

Comment:
Removing this will make the audit check happy.

Reply:
:facepalm: